packaging/linux/snap/package.mak

```diff
@@ -0,0 +1,10 @@
+snap:
+	snapcraft prime
+	cp ../../../icons/shotcut-logo-64.png gui
+	cp ../org.shotcut.Shotcut.desktop gui/shotcut.desktop
+	desktop-file-edit --set-key=Icon --set-value='${SNAP}/meta/gui/shotcut-logo-64.png'
+	snapcraft snap
```

Comment:
I have to use [cleanbuild](https://docs.snapcraft.io/reference/snapcraft-command#cleanbuild). Is there a way to support these additional commands? I could alter my process to use a docker container instead of cleanbuild. I will give that a try before deciding.